style.css

```diff
@@ -30,29 +32,130 @@ section {
 .first {
   color: rgba(255, 255, 255, 0.3);
   background: #50233B;
-  position: relative;
-  display: flex;
-  justify-content: flex-end;
-  align-items: flex-end;
-  overflow: hidden;
+  overflow: visible;
+}
+
+.first::before {
+  content: '';
+  position: absolute;
+  width: 12vh;
+  height: 12vh;
+  border-radius: 5%;
+  background: #50233B;
+  left: 10vh;
+  bottom: -5vh;
+  transform: rotate(45deg);
+  z-index: 1;
 }
 
 .second {
   color: rgba(255, 255, 255, 0.3);
   background: #FF9D00;
-  position: relative;
-  display: flex;
-  justify-content: flex-end;
-  align-items: flex-end;
-  overflow: hidden;
+}
+
+.second__image {
+  height: 40%;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  clip-path: polygon(20% 0%, 0% 20%, 30% 50%, 0% 80%, 20% 100%, 50% 70%, 80% 100%, 100% 80%, 70% 50%, 100% 20%, 80% 0%, 50% 30%);
```

Comment:
very nice idea :)

Reply:
Thanks, one of my favorite tricks out there. :)